client/components/main/index.jsx

```diff
@@ -11,7 +11,7 @@ module.exports = React.createClass( {
 
 	render: function() {
 		return (
-			<main className={ classnames( this.props.className, 'main' ) } role="main">
+			<main className={ classnames( this.props.className, 'main' ) } role="main" style={ this.props.style }>
```

Comment:
To be honest, I have no idea. @adambbecker put this there originally. I need to research this further. Thanks.